packages/core/admin/admin/src/content-manager/history/components/VersionHeader.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+
+import { HeaderLayout, Typography } from '@strapi/design-system';
+import { Link } from '@strapi/design-system/v2';
+import { ArrowLeft } from '@strapi/icons';
+import { Contracts } from '@strapi/plugin-content-manager/_internal/shared';
+import { useIntl } from 'react-intl';
+import { NavLink } from 'react-router-dom';
+
+import type { FormattedLayouts } from '../../utils/layouts';
+
+Contracts.ContentTypes;
+
+interface VersionHeaderProps {
+  headerId: string;
+  version: Contracts.HistoryVersions.HistoryVersionDataResponse;
+  layout: FormattedLayouts;
+}
+
+export const VersionHeader = ({ headerId, version, layout }: VersionHeaderProps) => {
+  const { formatMessage, formatDate } = useIntl();
+
+  const mainFieldValue = version.data[layout.contentType.settings.mainField];
+  const formattedSubtitle = mainFieldValue
+    ? formatMessage(
+        {
+          id: 'content-manager.history.version.subtitle',
+          defaultMessage: '{hasLocale, select, true {{subtitle}, in {locale}} other {{subtitle}}}',
+        },
+        {
+          hasLocale: Boolean(version.locale),
+          subtitle: `${mainFieldValue} (${layout.contentType.info.singularName})`,
+          locale: version.locale?.name,
+        }
+      )
+    : null;
```

Comment:
I wouldn't show _nothing_ if there's no mainFieldValue, just not the mainFieldValue. The locale and content type name are still interesting

Reply:
oh sorry I misunderstood you on slack 🤣 . My only concern is the content type name is wrapped in parentheses. Doesn't (article) or (article), in English (en) look weird...

Reply:
Yes we would need to remove the parenthesis in this case. It's probably going to get awkward in react-intl syntax though.

I guess you could have 2 translation keys, and use the result of one formatMessage() in another formatMessage?

Reply:
actually it's probably not worth it, this is an unlikely edge case